document/CoreFormat.mdk

```diff
@@ -2050,58 +2054,51 @@ which encodes the function signature in a manner that is compiler-dependent and
 ~
 
 
-If the `logicalLocationComponent` object occurs as the only member of an array within
-the `run.logicalLocations` object (&sect;[#run-logicalLocations]),
+If the `logicalLocation` object describes a top-level logical location,
 and if the `name` property would be equal to the name of the corresponding property, then the `name` property may be absent.
 
 ~ IsoNumberedExample { @IsoNumberedExample=1 }
-In this example, the array of `logicalLocationComponent` objects has only one element, and `name` is omitted.
+In this example, the logical location is a top-level C++ function named `functionF`, and `name` is omitted.
 ```
 "logicalLocations": {
-  "functionF": [
-    {
-      "kind": "function"
+    "functionF": {
+        "kind": "function"
     }
-  ]
 }
 ```
 ~
 
 ~ IsoNumberedExample
-In this example, the array of `logicalLocationComponent` objects has only one element, and `name` is equal to the property name.
+In this example, the logical location is a top-level C++ function, and `name` is equal to the property name.
 ```
 "logicalLocations": {
-  "functionF": [
-    {
-      "name": "functionF",
-      "kind": "function"
+    "functionF": {
+        "name": "functionF",
+        "kind": "function"
     }
-  ]
 }
 ```
 ~
 
 ~ IsoNumberedExample
-In this example, the array of `logicalLocationComponent` objects has only one element,
+In this example, the logical location is a top-level C++ function,
 but `name` is not equal to the property name, so it cannot be omitted.
 ```
 "logicalLocations": {
-  "functionF-0": [
-    {
-      "name": "functionF",
-      "kind": "function"
+    "functionF-0": {
+        "name": "functionF",
+        "kind": "function"
     }
-  ]
 }
 ```
 ~
 
-### `kind` property { #logicalLocationComponent-kind }
+### `kind` property { #logicalLocation-kind }
 
-A `logicalLocationComponent` object should contain a property named `kind`, whose value shall be one of the following strings,
+A `logicalLocation` object should contain a property named `kind`, whose value shall be one of the following strings,
 if any of those strings accurately describes the construct identified by this object:
 
-*   `"function"`
+*   `"function"` 
 *   `"member"`
 *   `"method"`
```

Comment:
Fixed.

---
In reply to: [64121106](https://github.com/sarif-standard/sarif-spec/pull/213#discussion_r64121106) [](ancestors = 64121106)